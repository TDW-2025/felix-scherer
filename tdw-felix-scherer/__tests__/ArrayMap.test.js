import React from "react";
import { render } from "@testing-library/react";
import ArrayMap from "@/app/array-map/page";

describe("ArrayMap Component", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<ArrayMap />);
    expect(asFragment()).toMatchSnapshot();
  });
});
