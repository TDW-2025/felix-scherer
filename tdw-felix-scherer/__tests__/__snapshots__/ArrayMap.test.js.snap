// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`ArrayMap Component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="page"
  >
    <div>
      <div>
        Item: First item
      </div>
      <div>
        Item: Second item
      </div>
      <div>
        Item: Third item
      </div>
      <div>
        Item: Fourth item
      </div>
    </div>
    <a
      href="/"
    >
      <button>
        Homepage
      </button>
    </a>
  </div>
</DocumentFragment>
`;
